lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -135,7 +136,8 @@ CommandInterpreter::CommandInterpreter(Debugger &debugger,
       m_skip_lldbinit_files(false), m_skip_app_init_files(false),
       m_comment_char('#'), m_batch_command_mode(false),
       m_truncation_warning(eNoOmission), m_max_depth_warning(eNoOmission),
-      m_command_source_depth(0) {
+      m_command_source_depth(0),
+      m_transcript(std::make_shared<StructuredData::Array>()) {
```

Comment:
This looks weird. Simply use default constructor is fine: `m_transcript()`

Reply:
Doesn't the default ctor create an empty shared pointer? Here what we want is to create the object, thus `make_shared`.

Reply:
This is out-dated now. In the latest commit, the field is now a `StructuredData::Array`. The initializer has been removed.